sdk/communication/AzureCommunicationChat/Tests/Recordings/test_StartPushNotifications_ReturnsSuccess.json

```diff
@@ -0,0 +1,52 @@
+{
+  "name" : "test_StartPushNotifications_ReturnsSuccess",
+  "interactions" : [
+    {
+      "request" : {
+        "body" : {
+          "clientDescription" : {
+            "templateKey" : "AcsIos.AcsNotify_Chat_3.0",
+            "aesKey" : "0000000000000000B00000000000000000000000AES=",
+            "platform" : "iOS",
+            "platformUIVersion" : "3619\/0.0.0.0\/",
+            "appId" : "AcsIos",
+            "authKey" : "0000000000000000B0000000000000000000000AUTH=",
+            "languageId" : "",
+            "cryptoMethod" : "0x70"
+          },
+          "transports" : {
+            "APNS" : [
+              {
+                "ttl" : 15552000,
+                "path" : "mockDeviceToken",
+                "context" : ""
+              }
+            ]
+          },
+          "registrationId" : "0E0F0BE0-0000-00C0-B000-A00A00E00BD0",
+          "nodeId" : ""
+        },
+        "url" : "https:\/\/edge.skype.com\/registrar\/prod\/v2\/registrations",
+        "headers" : {
+          "Content-Type" : "application\/json",
+          "User-Agent" : "azsdk-ios-AzureCommunicationChatTests\/1.0 (MacBookPro16,1 - 21.4.0; xctest:13.3 -> iOS 9.0; en_US)"
+        },
+        "method" : "POST"
+      },
+      "recorded_at" : 1652285387.352901,
+      "response" : {
+        "body" : "",
+        "url" : "https:\/\/edge.skype.com\/registrar\/prod\/v2\/registrations",
+        "status" : 202,
+        "headers" : {
+          "Strict-Transport-Security" : "max-age=31536000; includeSubDomains",
+          "ms-cv" : "Ll2sT+wUV0Clsg\/+DCI4bg.0",
+          "Content-Length" : "0",
+          "x-msedge-ref" : "Ref A: A0A998A78D6545FF83FE44E2FBCB47F5 Ref B: YVR311000109027 Ref C: 2022-05-11T16:09:47Z",
```

Comment:
What is this and does it need to be scrubbed?

Reply:
These are included in the http response from registrar service.

Reply:
That doesn't really answer the question. Is this potentially sensitive information? Should it be scrubbed? If your answer is "I don't know" then it probably should be.

Reply:
https://github.com/Azure/azure-sdk-for-ios/wiki/Writing-End-to-End-Tests-using-AzureTest#scrubbing-values

The skypeToken came from a header I believe, but for properties in the body, AzureTest has mechanisms to scrub properties. If the traffic is being recorded (which it clearly is) then you should be able to scrub it.

The question as to how to define "potentially sensitive information" is beyond this repo. It's a company-wide thing:
https://microsoft.sharepoint.com/sites/Privacy

But generally, things like that look like GUIDs or random strings are immediately suspect as being potentially identifiable or credentials, which is why I'm asking the question.

Reply:
Confirming with Registrar Service on this question.

Reply:
If you are asking "personally identifiable information", or something like personal credential, I think they are not.

**"MS-CV":** "rrksno7TNUa3NalS0pmU4g.0" This is like the unique identifier of a http request, like a requestId. We always use this ID to find and debug a specific request. I believe it has nothing to do with personal credential.

**"Strict-Transport-Security":** "max-age=31536000; includeSubDomains" 
This is apparently not something related to personal credential.

**"x-msedge-ref" :** "Ref A: 6126D5EF106A436B947B8DE148DB1754 Ref B: CO1EDGE1717 Ref C: 2022-05-12T20:32:08Z"
For this one, I noticed if we sent message to ACS resource, we will receive a "x-azure-ref" as well. I think this is more of a platform related identifier instead of a personal credential identifer.
If we try to send a "CreateThread" request to ACS, we will receive something like "x-azure-ref" : "0x259YgAAAAAFwxNuwWX4Q6CO2AGRkPldWVZSMzExMDAwMTE1MDE5ADlmYzdiNTE5LWE4Y2MtNGY4OS05MzVlLWM5MTQ4YWUwOWU4MQ=="
And this is also recorded in one of our recording files called "test_CreateThread_WithoutParticipants".

Reply:
Confirming with Registrar Service as you suggested.

Reply:
Please see the reply of Jaroslav Franek from Registrar Team -

**MS-CV** is definitely not Personal Data, it is System Metadata

**x-msedge-ref** is AFD/MIRA added header, which they claim can be copied to logs, so I would say System Metadata as well